README.md

```diff
@@ -33,6 +33,8 @@ ROOT_PREFIX: /applist
 
 GA_TRACKING_ID: Google Analytics Tracking ID
 
+AW_TRACKGA_TRACKING_ID: Google Ads Tracking ID
```

Comment:
is this a typo? It's `AW_TRACKING_ID` everywhere else.

Reply:
totally a type!

Reply:
@briehl It's fixed now.